### PR TITLE
enable IP addresses in SAN

### PR DESCRIPTION
**Pull request by dinghram**

### CODE REVIEW

**Review by kairoaraujo — APPROVED** [review]

**Review by necheffa — APPROVED** [review]
LGTM

File: ca.go [hunk 1]
```diff
@@ -32,6 +33,7 @@ type Identity struct {
 	Province           string   `json:"province" example:"Veldhoven"`                           // Province name
 	EmailAddresses     string   `json:"email" example:"sec@company.com"`                        // Email Address
 	DNSNames           []string `json:"dns_names" example:"ca.example.com,root-ca.example.com"` // DNS Names list
+	IPAddresses        []net.IP `json:"ip_addresses,omitempty" example:"127.0.0.1,192.168.0.1"` // IP Address list
```
Comment (ZenkieBear) [inline ca.go:36]: !!! The comment `example:"127.0.0.1,192.168.0.1"` is imcompatible with `[]net.IP`.
The effect is, when I generate the  swaggo docs will skip the model.Identity, and print an error on the console.

**Review by ZenkieBear — COMMENTED** [review]
Please fix the problem.
Don't give it default value , or write a temporary struct to convert **string** to `net.IP`

File: ca.go (hunk 1 above)
Comment (dinghram) [reply ca.go:36]: Good catch. I have put in a PR to remove the example: https://github.com/kairoaraujo/goca/pull/37

**Pull request merged**